dotcom-rendering/src/web/components/Picture.tsx

```diff
@@ -111,25 +97,72 @@ const getSizes = (
 	) {
 		// Showcase main media images (which includes numbered list articles) appear
 		// larger than in body showcase images so we use a different set of image sizes
-		return `(min-width: ${breakpoints.wide}px) 1020px, (min-width: ${breakpoints.leftCol}px) 940px, (min-width: ${breakpoints.tablet}px) 700px, (min-width: ${breakpoints.phablet}px) 660px, 100vw`;
+		if (breakpoint >= breakpoints.wide) return 1020;
+		if (breakpoint >= breakpoints.leftCol) return 940;
+		if (breakpoint >= breakpoints.tablet) return 700;
+		if (breakpoint >= breakpoints.phablet) return 700;
+		return breakpoint;
 	}
 
 	switch (role) {
 		case 'inline':
-			return `(min-width: ${breakpoints.phablet}px) 620px, 100vw`;
+			if (
+				breakpoint >= breakpoints.tablet &&
+				breakpoint < breakpoints.desktop
+			)
+				return 680;
+			if (breakpoint >= breakpoints.phablet) return 620;
+			return breakpoint; // 100vw
 		case 'halfWidth':
-			return `(min-width: ${breakpoints.phablet}px) 300px, 50vw`;
+			if (breakpoint >= breakpoints.phablet) return 300;
+			return breakpoint / 2; // 50vw
 		case 'thumbnail':
-			return '140px';
+			return 140;
 		case 'immersive':
-			return `(min-width: ${breakpoints.wide}px) 1300px, 100vw`;
+			if (breakpoint >= breakpoints.wide) return 1300;
+			return breakpoint; // 100vw
 		case 'supporting':
-			return `(min-width: ${breakpoints.wide}px) 380px, 300px`;
+			if (breakpoint >= breakpoints.wide) return 380;
+			if (breakpoint >= breakpoints.tablet) return 300;
+			return breakpoint; // 100vw
```

Comment:
Once under the tablet breakpoint, this is 100vw instead of 300px as it was before